WotC material/v13/pub_20170919_ToA.js

```diff
@@ -3,13 +3,85 @@ RequiredSheetVersion(12.999);
 // This file adds the backgrounds and beasts from the Tomb of Annihilation adventure book to MPMB's Character Record Sheet
 
 // Define the source
-SourceList.ToA={

```

Comment:
Why was the source removed?

Reply:
Woops! I was copying and pasting after moving stuff from the auto-gen files to the pub_ file and thought I had included the updated comment to the source.

Just pushed the fix.